server.js

```diff
@@ -13,13 +13,21 @@ let createUserDb = (username, password, leaning, email) => {
 };
 
 let getUserDb = (id) => {
-  return db.query(`SELECT * from users where userid = ${id}`);
+  return db.query(`SELECT * from users where articleid = ${id}`);
```

Comment:
This return path links the article id to the user id?

Reply:
updated - you caught it before I pushed up the fix:-p